react/LoadMore/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect, useRef, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import Button from '../Button'
+import Spinner from '../Spinner'
+
+const LoadMore = ({ fetchMore, label }) => {
+  const [isLoading, setIsLoading] = useState(false)
+  const elementRef = useRef()
+
+  const startFetchMore = useCallback(async () => {
+    if (!isLoading) {
+      setIsLoading(true)
+      await fetchMore()
+      setIsLoading(false)
```

Comment:
I considered adding a try/catch here… but we wouldn't be able to do anything with the error except maybe logging it. I think it's better to let it propagate.